src/devices/Si7021/Si7021.cs

```diff
@@ -0,0 +1,63 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.I2c;
+
+namespace Iot.Device
+{
+    public class Si7021 : IDisposable
+    {
+        const byte si7021_cmd_read_temperature = 0xE3;
```

Comment:
Constants should begin with an upper case. Also, I would have this be something like `ReadTemperatureCommand` instead